site/content/en/blog/releases/1.26.0-rc.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "1.26.0 - Release Candidate"
+linkTitle: "1.26.0-rc"
+date: "2022-09-06"
+---
+This is the 1.26.0-rc release of Agones.
+This release brings us support for Kubernetes 1.23, so if you are installing this release, remember to upgrade your clusters.
```

Comment:
Only thing I found, add a blank line between, so there is a line break between these two sentences.

```suggestion
This is the 1.26.0-rc release of Agones.

This release brings us support for Kubernetes 1.23, so if you are installing this release, remember to upgrade your clusters.
```

![image](https://user-images.githubusercontent.com/298370/188750639-537df7fb-bd70-45e7-9d93-468cb3294aee.png)


Otherwise, LGTM!